core/src/main/java/org/apache/druid/java/util/common/parsers/TimestampParser.java

```diff
@@ -87,7 +87,7 @@
       };
     } else {
       try {
-        final DateTimes.UtcFormatter formatter = DateTimes.wrapFormatter(DateTimeFormat.forPattern(format));
+        final DateTimes.Formatter formatter = DateTimes.wrapFormatter(format, userTimeZone);
```

Comment:
Any reason this isn't calling `DateTimeFormat.forPattern` on the `format` argument like it was previously?

Reply:
I've changed implement of DateTimes.wrapFormatter, added a param timezone. And wrapper the DateTimeFormat.forPattern into the class DateTimes$Formatter. @clintropolis